source/target/realtek/rtl8195am/target_reset.c

```diff
@@ -0,0 +1,55 @@
+/* CMSIS-DAP Interface Firmware
+ * Copyright (c) 2015-2017 Realtek Semiconductor Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <string.h>
+#include "target_reset.h"
+#include "swd_host.h"
+#include "DAP_Config.h"
+
+void target_before_init_debug(void)
+{
+    return;
+}
+
+uint8_t target_unlock_sequence(void)
+{
+    return 1;
+}
+
+uint8_t target_set_state(TARGET_RESET_STATE state)
+{
+    return swd_set_target_state_hw(state);
+}
+
+uint8_t security_bits_set(uint32_t addr, uint8_t *data, uint32_t size)
+{
+    return 0;
+}
+
+void swd_set_target_reset(uint8_t asserted)
+{
+    (asserted) ? PIN_nRESET_OUT(0) : PIN_nRESET_OUT(1);
+    if (asserted) {
+        swd_write_word(0xE000ED0C, 0x05FA0004);
```

Comment:
Wanted to keep this for future platform, but you're right, it's redundant here. Commit is updated.